Source/Blazorise/wwwroot/dragDrop.js

```diff
@@ -20,6 +20,38 @@ export function destroy(element, elementId) {
     element.removeEventListener('dragstart', dragStartHandler);
 }
 
+export function InitializeThrottledDragEvents(element, elementId, dotnetAdapter)
```

Comment:
We will also need `destroyThrottledDragEvents`.

Reply:
PS. name JS methods with first letter lowecased.

Reply:
Oops, it was late copy paste fail. :D